src/activities/activities.txt

```diff
@@ -10,10 +10,10 @@ align4
 align4-2players
```

Comment:
On a fork, you should never work on master directly. It will be a mess when you try to rebase with upstream master and a lot of conflicts.
Always keep the master as much as possible in sync with upstream master and do branches for development.